_config.yml

```diff
@@ -10,13 +10,13 @@ sass:
 
 # Resume settings
 resume_avatar:                  "true"
-resume_name:                    "Skatetocat"
-resume_title:                   "Professional Skater"
-resume_contact_email:           "skatetocat@github.com"
+resume_name:                    "GitHub Teacher"
+resume_title:                   "Professional Trainer"
+resume_contact_email:           "services@github.com"
```

Comment:
These new additions shouldn't over write the previous one, it should  be commented out.